src/report/writer.rs

```diff
@@ -94,6 +99,17 @@ fn log_line_info<O: Write + Send>(errors: &Errors, output: &mut O, indentation:
     _ = writeln!(output, "{}", ANSIStrings(line_info));
 }
 
+/// Log the additional number of this error that were found in other locations
+fn log_count<O: Write + Send>(errors: &Errors, output: &mut O, indentation: usize, count: usize) {
+    let line_count: &[ANSIString<'static>] = &[
+        errors.styles.style(Styled::Default).paint(format!("{:width$}", "", width = indentation)),
+        errors.styles.style(Styled::Location).paint("-->"),
+        errors.styles.style(Styled::Default).paint(" "),
+        errors.styles.style(Styled::Location).paint(format!("And {count} other locations")),
```

Comment:
"And" should be lowercase to match tiger's style (and because it's a sentence fragment)